src/lib/chips/chip-input.ts

```diff
@@ -124,8 +146,18 @@ export class MatChipInput {
   /** Checks to see if the (chipEnd) event needs to be emitted. */
   _emitChipEnd(event?: KeyboardEvent) {
     if (!this._inputElement.value && !!event) {
-      this._chipList._keydown(event);
+      const isRtl = this._dir && this._dir.value === 'rtl';
+      const isPreviousKey = event &&
+        (event.keyCode === LEFT_ARROW && !isRtl) ||
+        (event.keyCode === RIGHT_ARROW && isRtl);
```

Comment:
Doesn't the key manager handle the previous key already?

Reply:
No. The active item index is -1.